doc/changes/devel/11522.enhancement.rst

```diff
@@ -0,0 +1 @@
+Add a `colorbar` parameter to `mne.viz.plot_topomap()` to toggle the display of a colorbar, improving plot clarity.
```

Comment:
surrounding with single-backticks won't work to format as code (this is rST, not markdown). `colorbar` should get double-backticks, and the crossref should get an explicit `:func:` role (see other changelog entries in this folder for examples).

Also, all changelog entries should include the PR author and PR number (again, see other changelog entries in this folder for examples).